Add english translation for analysis page.

FabioFuschi-SNK opened the pull request:
This PR adds the English translation for the analysis page.

MikeMcC399 commented:
The app and the FAQs use US English spelling, so I'm not sure if that should apply to the spelling in the analysis page.

"behaviour" is British English, whereas "behavior" would be US English.

MikeMcC399 commented on src/data/analyse_en.json:
```diff
@@ -1,4 +1,314 @@
+    	},
+    	{
+    		"title": "BMG: Impfdashboard (EN)",
+            "desc": "Workdaily updated information on vaccination status and progress in Germany, from the website 'together against corona' from the German federal ministry of health (BMG).",
```
"Workdaily" should probably be "Daily", then it would match the other titles.

MikeMcC399 commented on src/data/analyse_en.json:
```diff
@@ -1,4 +1,314 @@
+    		"link": "https://impfdashboard.de/en/"
+    	},
+    	{
+    		"title": "WHO: COVID 19 Dashboard (EN)",
```
Suggest
"COVID 19" => "COVID-19"

to match the text on the website https://covid19.who.int/ itself.

MikeMcC399 commented on src/data/analyse_en.json:
```diff
@@ -1,4 +1,314 @@
+            "title": "Sharing behaviour",
+            "export": "https://obs.eu-de.otc.t-systems.com/obs-public-dashboard/csv/cwa_sharing_data.zip",
+            "info": {
+                "content": "<strong>Sharing behaviour</strong><p>Count of positive test results, for which a QR code was registered or a teleTAN requested for validation, as well as the sum of positive test results validated with either option.</p><strong>Sharing behaviour: QR codes</strong><p>Count of positive test results, validated through QR code registration, shared and not shared in the Cor ...
```
"Corona warning app" => "Corona-Warn-App"

MikeMcC399 commented on src/data/analyse_en.json:
```diff
@@ -1,4 +1,314 @@
+    "updateNotice": "The data of the CWA dashboard is updated on a daily basis at 10:30 AM.",
+    "range":{
+        "picker": {
+            "timerange": "Timerange",
```
"Timerange" => "Time range"

MikeMcC399 commented on src/data/analyse_en.json:
```diff
@@ -1,4 +1,314 @@
 {
-    "lang": "en-US",
-    "documentLang": "en"
+    "lang": "en-UK",
```
Shouldn't this be "en-GB"?

MikeMcC399 reviewed (changes requested):
Everything works!

I just made a few comments suggesting text changes.

I noticed that the date appears in US format, not European English format with 

dd/mm/yyyy.

Maybe this is linked to the lang definition which is set to "en-UK"?

FabioFuschi-SNK commented:
> Everything works!
> 
> I just made a few comments suggesting text changes.
> 
> I noticed that the date appears in US format, not European English format with
> 
> dd/mm/yyyy.
> 
> Maybe this is linked to the lang definition which is set to "en-UK"?

@MikeMcC399 Thank you for the hint, i changed the date format to GB date format now. 

I have also changed the wording of the mentions in the English translation

MikeMcC399 commented:
@FabioFuschi-SNK 
Thank you for the changes! Would you like to mark each of them as resolved? (See https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request). That makes it easier for you, me and anybody else reading this PR.

MikeMcC399 commented on src/data/analyse_en.json:
```diff
@@ -1,4 +1,314 @@
+    	},
+    	{
+    		"title": "BMG: Impfdashboard (EN)",
+            "desc": "Daily updated information on vaccination status and progress in Germany, from the website 'together against corona' from the German federal ministry of health (BMG).",
```
"federal ministry of health" should be capitalized:

Federal Ministry of Health

Reference:
https://www.bundesgesundheitsministerium.de/en/legal-notice.html

MikeMcC399 reviewed (changes requested):
The Federal Ministry of Health (Bundesministerium für Gesundheit, BMG)

should be capitalized.

MikeMcC399 reviewed (approved):
@FabioFuschi-SNK 

This looks good to publish now! 👍🏻

MikeMcC399 commented:
See also https://github.com/corona-warn-app/cwa-website/issues/2091 and for `/en/analysis/`:

"Last data update: November 17, 2021, 9:21 AM GMT+1. The data of the CWA dashboard is updated on a daily basis at 10:30 AM."

I don't think that this minor issue about the timestamp differences should stop the English page being published though.

dsarkar commented:
@FabioFuschi-SNK Many thanks for providing the PR.
@MikeMcC399  Many thanks for the review!

MikeMcC399 commented:
Now listed on Twitter:
https://twitter.com/coronawarnapp/status/1461311095688290309?s=20 🙂
